browser/components/TableApprovalCell.tsx

```diff
@@ -35,7 +35,7 @@ export default class TableApprovalCell extends React.Component<Partial<Props>, S
       <Cell {...props}>
         <div className="row">
           <div className="col-md-6">
-            <Link to={`/approvals/${id}`}>Approve/Deny</Link>
+            <Link className="btn btn-info im-blue" to={`/approvals/${id}`}>Approve/Deny</Link>
```

Comment:
What is `im-blue` and the other `im-color` classes here and in other files?

Reply:
I thought it was a name of a coloring option via bootstrap but now I am not certain. I will double check to see if I was wrong but will remove otherwise.